web/src/component/community/Ranking/DrinkRanking.tsx

```diff
@@ -0,0 +1,104 @@
+import ContentTitle from "../ContentTitle";
+
+export default function DrinkingRanking() {
+  return (
+    <div className="mx-auto w-inner pt-30">
+      <div className="h-full w-full rounded-20 bg-white p-20">
+        <ContentTitle
+          title={"🎁 가장 많은 음료를 받은 사람은!"}
+          subTitle={"오후 12시 기준"}
+        />
+        <div>
+          <div className="mb-10 grid grid-cols-3 items-end gap-14">
+            <div className="relative flex h-140 w-full items-center justify-center rounded-10 bg-green">
+              <div className="absolute top-10 left-5 h-24 w-24">
+                <img
+                  className="w-24 object-contain"
+                  src="../../src/assets/img/icon/second-prize.png"
+                  alt="1등"
+                />
+              </div>
+              <div className="content">
+                <div className="mb-15 h-45 w-45 rounded-full bg-white">
+                  {/* <img src="" alt="" /> */}
+                </div>
+                <div className="text-1 font-mediumtext-1 text-center font-medium text-white">
+                  <h3 className="mb-10">이름</h3>
+                  <p>00잔</p>
+                </div>
+              </div>
+            </div>
+            <div className="relative flex h-150 w-full items-center justify-center rounded-10 bg-green">
```

Comment:
보면서 느낀게 이렇게 작업을 하면 나중에 사용자가 여러명 들어올때 처리하기 어려울거 같아.

서버에서 데이터를 100개 심지어 몇만개 들어오면 일일이 사람 이름을 적을 수는 없으니까.

보통 프론트엔드에서는 이런 반복되는 부분을 컴포넌트로 만들고 재사용성을 추구해

지금은 내가 서버를 당장 만들 여력이 없어서 너가 임시로 데이터를 만들어서 적용하면 좋을거 같은데

```suggestion
const test_Data = ['a', 'b', 'c']

test_Data.map((name, rank, idx) => {
return <Your Component name={name}, idx={idx}, rank ={rank}/>
})
```

이런식으로 작업하는게 보통이야

내가 서버 작업이 매우 느려서 너도 이해가 잘 안될것 같지만 서버가 어떤 식으로 데이터를 주면 좋을지 생각하면서 코딩하면 좋을거 같아

Reply:
서버에서 데이터를 받아오는 파일을 하나 만들어서 그 파일 안에 있는 변수를 활용하려고 합니다!
`data.tsx` 파일 생성하고 적용하겠습니다!